pkg/interface/src/views/apps/chat/components/ChatWindow.tsx

```diff
@@ -182,6 +182,7 @@ export default class ChatWindow extends Component<ChatWindowProps, ChatWindowSta
     if (this.props.unreadCount === 0) return;
     this.props.api.chat.read(this.props.station);
     this.props.api.hark.readIndex({ chat: { chat: this.props.station, mention: false }});
+    this.props.api.hark.readIndex({ chat: { chat: this.props.station, mention: true }});
```

Comment:
Why do we dismiss mention notifications in a different way than regular ones?

Reply:
Just a peculiarity of the API I guess. They're not merged with regular notifications, so we need to address them differently.